setup.py

```diff
@@ -34,18 +35,26 @@ def get_extras_require():
         "tools",
         "all-contrib-expectations",
     )
+
+    # Use Path() from pathlib so we can make this section of the code OS agnostic.
     requirements_dir = "reqs"
-    rx_fname_part = re.compile(rf"{requirements_dir}/requirements-dev-(.*).txt")
-    for fname in glob(f"{requirements_dir}/*.txt"):
-        match = rx_fname_part.match(fname)
+    glob_path = Path(f"{requirements_dir}/*.txt")
+    rx_name_part = re.compile(rf"requirements-dev-(.*).txt")
+
+    # Loop through each requirement file and verify they are named
+    # correctly and are in the right location.
+    for glob_file_name in glob(f"{glob_path}"):
```

Comment:
If we are switching to using `pathlib` for this (which I strongly agree with 😄 ), we do the `glob` operation directly from the `pathlib.Path` object.
https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob